### ISSUE TITLE
Query for substances can include experiences

### ISSUE THREAD

**Issue by d6y** [psychonautwiki/bifrost#41]
When using the substance query, I observed a record that appears to be an experience. 

I'd expect only substances to be returned from the substance query.

You can find the experience I observed with this query:

```
 substances(query: "Experience:3-MeO-PCP, LSD, Clonazolam, and Amphetamine - Excessive Amounts and Excessive Confusion") {
        name 
    	url
    }
```

**Comment by 19h** [comment]
Are they also included in the limit query?

**Comment by d6y** [comment]
@19h yes - that's how I noticed it. The `substances(limit: 999)` query included that experience as one of the results.
